# --- T2-COPYRIGHT-NOTE-BEGIN ---
# This copyright note is auto-generated by ./scripts/Create-CopyPatch.
# 
# T2 SDE: package/.../linux24-psionw/revo-battery.patch
# Copyright (C) 2004 - 2005 The T2 SDE Project
# 
# More information can be found in the files COPYING and README.
# 
# This patch file is dual-licensed. It is available under the license the
# patched project is licensed under, as long as it is an OpenSource license
# as defined at http://www.opensource.org/ (e.g. BSD, X11) or under the terms
# of the GNU General Public License as published by the Free Software
# Foundation; either version 2 of the License, or (at your option) any later
# version.
# --- T2-COPYRIGHT-NOTE-END ---

The new battery driver was written with Series 5 meassurements in mind - so
here I add some guess first-shoot values for the Revo familly.

  - Rene Rebe <rene@exactcode.de>

--- linux-2.4.27/drivers/ssi/ads7843.c.vanilla	2005-02-07 10:38:46.864858176 +0100
+++ linux-2.4.27/drivers/ssi/ads7843.c	2005-02-07 10:45:28.858745816 +0100
@@ -98,10 +98,17 @@
 
 
 //Assume default NiMh battery
+#if defined(CONFIG_PSIONW_REVO) || defined(CONFIG_PSIONW_REVOPLUS)
+static int bat_max_voltage = 3600; //[mV]
+static int bat_min_voltage = 2800; //[mV]
+static int bat_capacity = 800; //[mAh]
+static int bat_avg_current = 20; //[mA]
+#else
 static int bat_max_voltage = 2600; //[mV]
 static int bat_min_voltage = 2400; //[mV]
 static int bat_capacity = 1300; //[mAh]
 static int bat_avg_current = 100; //[mA]
+#endif
 
 #define BAT_OFF 1800 //1.8V psion is switched off
 
